### PR TITLE
Math Magician Website Testing

### PR DESCRIPTION
**Pull request by Godswilly**
In this PR, 
-  I updated the site with the necessary routes and added more styling.
-  I added react-testing modules and performed testing on the calculator app and snapshot tests.

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const NavBar = () => (
+  <nav className="navbar">
+    <h1>Math Magicians</h1>
+    <ul className="navbar-menu">
+      <li><a href="/">Home</a></li>
+      <li><a href="/App">Calculator</a></li>
+      <li><a href="/Quotes">Quotes</a></li>
```
Comment (teekaytech) [inline src/components/NavBar.js]: Since you've already used React router to set up the pages you're expected to use the React Router's `<Link>` tag, not the `<a>` tag.
Please try to update these changes in all other places that similar cases might occur.

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    <NavBar />
+    <h2>Welcome</h2>
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipisicing elit,
+      sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+      enim ad minim veniam, quis nostrud exercitation ullamco laboris
+      nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in
+      reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+      pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+      culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
```
Comment (teekaytech) [inline src/components/Home.js]: Replace with real text.

**Review by teekaytech — CHANGES_REQUESTED** [review]
## Status: Changes Required :warning: :vertical_traffic_light: 

### Hello @Godswilly :wave:

__I must commend your efforts so far in this project. Here are the major highlights for me:__

- You've implemented almost all the required features of the project :+1: 
- All linter checks are running/passing successfully :100: 
- PR title, summary, and commit messages looks awesome :+1: 

__However, some changes are required before this project can be approved. Please check my comments below and fix them as appropriate.__

**[ REQUIRED CHANGES ]**

- [ ] For the calculator page, you're expected to have some text to the left of the page, then the calculator to the right,  just as specified in the PDF file.
scren

- [ ] Kindly add more test cases:
  - [ ] It will be better if you can have a separate test file for the `calculate.js` file.
  - [ ] Double-check to ensure that all functions in the `calculate.js` file are covered with unit tests
  - [ ] Please add test cases to both positive and negative scenarios.
  - [ ] Add a snapshot test to ensure that the calculator page is rendering.

- [ ] Check my inline comments below and fix them as appropriate.

**[ OPTIONAL CHANGES ]**

- [ ] It will be nice if you can have a some meaningful texts on the homepage. Having `lorem ipsum` text is not really professional. 


* * *
**Please make the required changes and submit another review request when you are done.**

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

* * *

**Taofeek Olalere
[Twitter](https://twitter.com/ola_lere) | [Github](https://github.com/teekaytech) | @taofeekolalere on Slack**
__Happy Coding :smiley:__

**Review by arslanbisharat — APPROVED** [review]
# Approved :clap:

Hello Developer :wave:

Thank You for submitting this Pull Request. I am thrilled to inform you that your project is approved now. :clap:

You have done a great job. I like your project. 

**To highlight** 

- [ ] You have a descriptive Readme. Nice Job. 
- [ ] Your Pull Request is nice. Good Job!
- [ ] You have deployed your website. Nice!!!
- [ ] You are using linters and no error is present. I am happy to see it. 
- [ ] Your project has met the minimum technical requirements too. 



In short, I am very excited to see this project and proud of you. 

**What to do next?** :thinking:

1- Visit https://dashboard.microverse.org/projects and submit your approved project. 
2- Prepare yourself for the next project. 

I am wishing you very good luck with the next project. :smile:

Once again, Congratulations!:joy:

**Best,
Arslan Bisharat(PT-TSE)
Slack: @arslanbisharat**
